src/main/java/com/mparticle/kits/BranchMetricsKit.java

```diff
@@ -174,14 +200,20 @@ public void onInitFinished(JSONObject jsonResult, BranchError branchError) {
             getKitManager().onError(error);
         }
     }
-
+    
     @Override
     public void onApplicationForeground() {
         Branch.getAutoInstance(getContext().getApplicationContext(), getSettings().get(BRANCH_APP_KEY)).initSession(this);
     }
-
+    
     @Override
     public void onApplicationBackground() {
-
-    }
+    
+    }
+    
+    
+    // Region Branch event translation methods
+    
+    
+    // end Region
```

Comment:
please remove these 2 comments, since it looks like this logic ended up in the BranchUtil class